### PR TITLE
[cleanup] pegasus,marian,mbart pytorch tests

### PR DESCRIPTION
**Pull request by sshleifer**
+ Cleans up 3 pytorch test files
+ Faster (num_beams=2) pegasus integration test.

### CODE REVIEW

File: tests/test_modeling_mbart.py [hunk 1]
```diff
@@ -91,32 +90,6 @@ class MBartEnroIntegrationTest(AbstractSeq2SeqIntegrationTest):
     ]
     expected_src_tokens = [8274, 127873, 25916, 7, 8622, 2071, 438, 67485, 53, 187895, 23, 51712, 2, EN_CODE]
 
-    @slow
-    @unittest.skip("This has been failing since June 20th at least.")
```
Comment (sgugger) [inline tests/test_modeling_mbart.py:95]: I have zero context on why you're removing that test, if you could just explain?

**Review by sgugger — APPROVED** [review]
Adjustments look good.

File: tests/test_modeling_mbart.py (hunk 1 above)
Comment (sshleifer) [reply tests/test_modeling_mbart.py:95]: It has never really been passing, I don't even remember why I checked it in.

Comment (sgugger) [reply tests/test_modeling_mbart.py:95]: Ok then, I guess

**Review by LysandreJik — APPROVED** [review]
LGTM!

**Pull request merged**